templates/etc/elasticsearch/jvm.options.d/jvm.options.erb

```diff
@@ -0,0 +1,4 @@
+### MANAGED BY PUPPET ###
```

Comment:
can you please provide this as epp template? we require that for new templates (it is the erb successor).

Reply:
Ok, i try to change erb in epp. I will rebase after corrections/modifications